lib/ansible/modules/database/postgresql/postgresql_user.py

```diff
@@ -46,16 +46,18 @@
   password:
     description:
     - Set the user's password, before 1.4 this was required.
-    - Password can be passed unhashed or hashed (MD5-hashed).
+    - Password can be passed unhashed or hashed (MD5-hashed or SCRAM-SHA256-hashed).
```

Comment:
Please, add info that scram method is available from 2.9

Reply:
Do we have to wait for 2.9? In the end, this PR does not change any interfaces, just fixing reapplying password on SCRAM encrypted installations.

I'll remove this from doc

Reply:
@gundalow ?

Reply:
This PR is listed as `Feature Pull Request`, if it's truely a bug fix please update that.

This will not go into 2.9.0, though it *could* go into 2.9.1

Reply:
@Andersson007 I've removed this mention, cause it's not a feature (and it's available right now, without any fix)

@gundalow Renamed the PR. May I backport it to 2.8?

Reply:
I've changed this to a bug fix.
1) Please add a changelog/fragment https://docs.ansible.com/ansible/devel/community/development_process.html#making-your-pr-merge-worthy
2) Once this PR has been merged into `devel` you an create a backport by following the process on https://docs.ansible.com/ansible/devel/community/development_process.html#backporting-merged-prs

Reply:
done